README.adoc

```diff
@@ -163,13 +163,24 @@ configuration management process.
 === Prerequisites
 
 * Install repo and docker if it is not already done.
+* Install make and pkg-config if t is not already done.
 
-On Ubuntu, please run:
+On Ubuntu 20.04:
```

Comment:
Except for installing package like make or pkg-config, I think it is better to have distribution-agnostic instructions.